bindings/elixir/lib/sqlite_vss.ex

```diff
@@ -0,0 +1,224 @@
+defmodule SqliteVss do
+  @moduledoc """
+  Documentation for `SqliteVss`.
+  """
+  use Application
+  require Logger
+
+  @doc false
+  def start(_, _) do
+    Supervisor.start_link([], strategy: :one_for_one)
+  end
+
+  @doc """
+  Returns the configured sqlite_vss version.
+  """
+  def configured_version do
+    cacertfile = cacertfile() |> String.to_charlist()
+
+    http_options =
+      [
+        ssl: [
+          verify: :verify_peer,
+          cacertfile: cacertfile,
+          depth: 2,
+          customize_hostname_check: [
+            match_fun: :public_key.pkix_verify_hostname_match_fun(:https)
+          ]
+        ]
+      ]
+
+    config = :hex_core.default_config()
+
+    updated_config =
+      Map.put(config, :http_adapter, {:hex_http_httpc, %{http_options: http_options}})
+
+    case :hex_api_package.get(updated_config, "sqlite_vss") do
+      {:ok, {200, _headers, response}} ->
+        response
+        |> Map.get("releases")
+        |> Enum.map(& &1["version"])
+        |> List.first()
+
+      {:ok, {404, _, _}} ->
+        # TODO: remove this fallback once sqlite_vss is published
```

Comment:
I added this here to get passed the chicken or the egg here.  Once its published this could be removed or left as a default.